src/sys/MEM.ASM

```diff
@@ -264,6 +295,77 @@ in_line:                                        ; Load a line manually
     ret                                         ; Return to caller
 
 
+; SHIFT I/O BUFFER: Shifts everything past one index in the buffer by a certain number of bytes (right by default)
```

Comment:
Regular `SHIFT` cannot be used for insertions/lines as it uses the old buffer paradigm.